packages/bierzo-wallet/src/communication/signing/index.ts

```diff
@@ -0,0 +1,44 @@
+/*global chrome*/
+import { JsonRpcRequest, parseJsonRpcResponse2, isJsonRpcErrorResponse } from '@iov/jsonrpc';
+import { TransactionEncoder } from '@iov/core';
+import { isPublicIdentity, PublicIdentity } from '@iov/bcp';
+import { ethereumCodec } from '@iov/ethereum';
+import { extensionId } from '..';
+
+const generateSigningRequest = (): JsonRpcRequest => ({
```

Comment:
The naming of this methods and the file is misleading: "getIdentities" will give you the user's identities (i.e. public keys and chain IDs). This is a read-only operation. No signing happens here.

Reply:
👌